lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -58,15 +55,20 @@ def self.localize_project
         def self.update_metadata
           Action.sh("cd #{ENV['PROJECT_ROOT_FOLDER']} && ./Scripts/update-translations.rb")
 
-          strings_files = Dir.chdir(File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'])) do
-            Dir.glob('*.lproj/*.strings')
-          end
           Fastlane::Helper::GitHelper.commit(message: 'Update translations', files: strings_files, push: false)
 
           Action.sh('cd fastlane && ./download_metadata.swift')
 
           Fastlane::Helper::GitHelper.commit(message: 'Update metadata translations', files: './fastlane/metadata/', push: true)
         end
+
+        def self.strings_files
+          Dir.glob(
+            File.join(
+              ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], '*.lproj', '*.strings'
+            )
```

Comment:
I don't think this will work because I think `*` might be interpreted literally/verbatim here (not entirely sure if the shell substitution that would interpret that `*` as a wildcard?). Also, as mentioned previously, won't work for all the repos.

Maybe this instead would cover all cases?
```suggestion
            Dir.glob(File.join(
              ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], '**', '*.lproj', '*.strings'
            ))
```

Reply:
Interestingly this has a conflict on WPiOS, but it appears to work the way we want 🙂

Reply:
@jkmassel what conflict are you seeing? 

I tried it just now (with @AliSoftware's `**` suggestion) and it seemed like it worked fine, as far as I can see. It definitely found and added the files: https://github.com/wordpress-mobile/WordPress-iOS/pull/16432.

Reply:
Just to update this - it was a conflict with some deprecated localizations – removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/16415 and everything worked great after that.